src/v/redpanda/admin_server.cc

```diff
@@ -2758,9 +2758,14 @@ admin_server::get_reconfigurations_handler(std::unique_ptr<ss::http::request>) {
             reconfiguration_states.error()),
           ss::http::reply::status_type::service_unavailable);
     }
+    // we are forced to use shared pointer as underlying chunked_fifo is not
```

Comment:
I'm wondering what is forcing a copy in the first place, all arguments are moved in the underlying stream function. smells like a missing `mutable` somewhere somewhere and the const ness if forcing a copy. 🤔